Tools/sitl_run.sh

```diff
@@ -105,8 +105,11 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 			fi
 		fi
 		SIM_PID=$!
-		gz model --spawn-file="${src_path}/Tools/sitl_gazebo/models/${model}/${model}.sdf" --model-name=${model} -x 1.01 -y 0.98 -z 0.83
 
+		if gz model --spawn-file="${src_path}/Tools/sitl_gazebo/models/${model}/${model}.sdf" --model-name=${model} -x 1.01 -y 0.98 -z 0.83 2>&1 | grep "An instance of Gazebo is not running."; then
```

Comment:
What's up with the magic coordinates? `-x 1.01 -y 0.98 -z 0.83`?

Reply:
@dagar That was something I added a while ago to spawn the vehicle somewhere that is not zero, and was specified in the world file previously.

Reply:
@Jaeyoung-Lim Remember I asked the same question (see note below)? Perhaps worth an in-source comment.

![image](https://user-images.githubusercontent.com/5368500/88237892-a2c91e80-ccc3-11ea-93b9-be1e5ef43a04.png)